mlflow/tracing/display/display_handler.py

```diff
@@ -1,16 +1,67 @@
+import html
 import json
 import logging
 from typing import TYPE_CHECKING
+from urllib.parse import urlencode, urljoin
 
+import mlflow
 from mlflow.environment_variables import MLFLOW_MAX_TRACES_TO_DISPLAY_IN_NOTEBOOK
 from mlflow.utils.databricks_utils import is_in_databricks_runtime
+from mlflow.utils.uri import is_http_uri
 
 _logger = logging.getLogger(__name__)
 
 if TYPE_CHECKING:
     from mlflow.entities import Trace
 
 
+TRACE_RENDERER_ASSET_PATH = "/static-files/lib/notebook-trace-renderer/index.html"
+
+IFRAME_HTML = """
+<div>
+  <style scoped>
+  button {{
+    border: none;
+    border-radius: 4px;
+    background-color: rgb(34, 114, 180);
+    font-family: -apple-system, "system-ui", "Segoe UI", Roboto, "Helvetica Neue", Arial;
+    font-size: 13px;
+    color: white;
+    margin-top: 8px;
+    margin-bottom: 8px;
+    padding: 8px 16px;
+    cursor: pointer;
+  }}
+  button:hover {{
+    background-color: rgb(66, 153, 224);
+  }}
+  </style>
+  <button
+    onclick="const display = this.nextElementSibling.style.display;
+const isCollapsed = display === 'none';
```

Comment:
nit: Are indents important here? Otherwise, can we format these js codes?

Reply:
i don't think the indents are important since JS ignores whitespace. let me format